internal/store/postgres/migrations/000014_add_indexes_to_improve_slow_queries.up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE INDEX IF NOT EXISTS idx_appeals_created_by_status_deleted_at_updated_at
+ON appeals (created_by, status, deleted_at, updated_at DESC)
```

Comment:
why are these fields indexed in a single index instead of creating individual index for each?

Reply:
In general, it's a trade-off between index size and query performance. An index that covers multiple fields will be larger and slower to update, but can improve query performance. An index on a single field will be smaller and faster to update, but may not improve query performance as much.

IMHO better to use multiple fields for this case